linkis-engineconn-plugins/spark/src/main/scala/org/apache/linkis/engineplugin/spark/launch/SparkSubmitProcessEngineConnLaunchBuilder.scala

```diff
@@ -115,8 +124,32 @@ class SparkSubmitProcessEngineConnLaunchBuilder(builder: JavaProcessEngineConnLa
       memory
     }
 
+    //    val deployMode: String = getValueAndRemove(properties, SPARK_DEPLOY_MODE)
+    val deployMode: String = SPARK_DEPLOY_MODE.getValue(properties)
```

Comment:
Should use this method ：
 val label = LabelUtil.getEngingeConnRuntimeModeLabel(engineCreationContext.getLabels())
    val isYarnClusterMode: Boolean =
      if (null != label && label.getModeValue.equals(LabelValueConstant.YARN_CLUSTER_VALUE)) true
      else false

Reply:
Ok, thanks for reviewing the code